python-backend/app/api/mines/reports/resources/mine_reports.py

```diff
@@ -81,14 +82,11 @@ def post(self, mine_guid):
             permit_id=permit.permit_id if permit else None)
 
         submissions = data.get('mine_report_submissions')
-        if submissions is not None:
-            report_submission_guid = uuid.uuid4()
+        if submissions:
             submission = submissions[0]
             if len(submission.get('documents')) > 0:
-                report_submission = MineReportSubmission(
-                    mine_report_submission_guid=report_submission_guid,
-                    mine_report_submission_status_code='MIA',
```

Comment:
Why do we need this status on MineReportSubmission; or - at least - why are we requiring it to be provided in this create?

Reply:
That's a good point, not sure the origin of that decision. @JustinMacaulay or @nayr974 ?